src/Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -10,13 +10,10 @@ file that was distributed with this source code.
 #}
 
 {% macro render_form(form, permissions, td_type, admin, admin_configuration, object) %}
-    <form class="form-horizontal"
```

Comment:
the class is missing

Reply:
This is a BC break / fix, depending on your application.

It *fixes* the ignored form theme of the `form`, but it now respects the theme you have set for your admin.